examples/integration/xcodeproj_targets.bzl

```diff
@@ -153,4 +156,23 @@ def get_xcode_schemes():
                 ],
             ),
         ),
+        xcode_schemes.scheme(
+            name = "iOSAppUnitTestSuite_Scheme",
+            test_action = xcode_schemes.test_action(
+                env = {
+                    "IOSAPPSWIFTUNITTESTS_CUSTOMSCHEMEVAR": "TRUE",
+                },
+                targets = [
+                    "//iOSApp/Test/ObjCUnitTests:iOSAppObjCUnitTestSuite",
+                    "//iOSApp/Test/SwiftUnitTests:iOSAppSwiftUnitTestSuite",
+                ],
+                post_actions = [
+                    xcode_schemes.pre_post_action(
+                        name = "Run After Tests",
+                        script = "echo \"Hi\"",
+                        expand_variables_based_on = "//iOSApp/Test/SwiftUnitTests:iOSAppSwiftUnitTests",
```

Comment:
Oops, this should probably be one of the `targets` passed above and not this. 

ps: a custom scheme for raw `test_suite` will be added as well but in a follow up (some remaining things to implement there in order to support it)

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/2196/commits/77f8c029649d0f4ebe037dc443d6595c40798df7